C++/Navio/Common/gpio.cpp

```diff
@@ -16,9 +16,10 @@
 #define HIGH                1
 
 /* Raspberry Pi GPIO memory */
-#define BCM2708_PERI_BASE   0x20000000
-#define BCM2709_PERI_BASE   0x3F000000
-#define BCM2835_PERI_BASE   0x3F000000
+
```

Comment:
Why is the blank line added? Let's remove it. Other Comments are not separated via blank lines.